### ISSUE TITLE
QA Standards - Authenticated Experience - Profile - Authenticated Menu

### ISSUE THREAD

**Issue by it-harrison** [department-of-veterans-affairs/va.gov-team#65194]
#### VFS team name
Authenticated Experience - Profile

#### VFS product name
Authenticated Menu

#### VFS feature name
Authenticated Menu Modifications

#### Point of Contact/Reviewers
@it-harrison  - Ian Harrison - QA
---
#### Regression Test Plan 
- [x] Standard has been met
- [ ] Standard has not been met
#### Test Plan
- [x] Standard has been met
- [ ] Standard has not been met
#### Traceability Reports 
- [x] Standard has been met
- [ ] Standard has not been met
#### E2E Test Participation
- [x] Standard has been met
- [ ] Standard has not been met
#### Unit Test Coverage
- Lines %:

- Functions %:

- Statements %:

- Branches %:

- [x] Standard has been met
- [ ] Standard has not been met
Note: there was no formal QA for this initiative and therefore no test coverage data was reported.
##### Explanation of failure to meet standard (if applicable): #####
---
### VFS Guidance

- Close the ticket when your Product Owner determines that you have sufficiently met the QA Standards.
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket

### Platform directions
- Update "Issue Title" to be of the form "QA Standards - VFS Team - VFS Product"
- Add the VFS team, product name, and feature name
- Add your name, practice area, and GH handle under Point of Contact/Reviewers
- Complete the QA Standards section, making sure to include an "Explanation of failure to meet standard" for every Standard the product does not meet.
- Link to the Collaboration Cycle Request epic
- Add assignees: 
  - VFS PM
  - Yourself (optional)
- Add labels:
  - VFS team label
  - VFS product label
  - QA-standards label
  - launch-blocking label if the product has failed to meet a required QA Standard

**Comment by mtcA6** [comment]
@it-harrison is this launch blocking? 

@adamwhitlock1 can you take a look at this and add points? Hoping we could pick this up in the next sprint, launch blocking or not.

**Comment by it-harrison** [comment]
@mtcA6 No, it is not launch-blocking. Only e2e tests are launch-blocking.

**Comment by adamwhitlock1** [comment]
Statements that are uncovered by unit tests are not from the work we added, and is from previous work by site wide codeowners. We will bring this up with that team, but wont be updating their tests at this point.

**Comment by mtcA6** [comment]
Not owned by auth-exp but sitewide team also seems unlikely to create the test coverage

**Comment by jilladams** [comment]
Sitewide Public Websites team delivery manager chiming in here. Am I understanding correctly that this: 
Unit Test Coverage
Lines %: 90
Functions %: 100
**Statements %: 71.42**
Branches %: 94.44

Is the problem? Is there someplace to see the test output to understand the gaps? If this is not launch blocking, between Travis's team and mine, we'll need to assess what's not covered, and prioritize accordingly.

**Comment by jilladams** [comment]
Also worth noting: the header is headed toward a rebuild that we'll be designing/researching in Q4, and and hoping to start build before EOY, so we may not prioritize unit tests for header code that will get overhauled under that project.

**Comment by jilladams** [comment]
Thanks @it-harrison . Where can we see how the report generated that #, in order to assess where we need to add coverage?

**Comment by adamwhitlock1** [comment]
Platform is working on making this report public and published with deployments, but you can run the report locally by running this command on vets-website:

``` bash 
yarn test:coverage src/platform/**/*.unit.spec.js* --coverage-html
```

That will take about a minute to run, and then it will spit out a whole coverage folder in the root of vets-website. If you open the `coverage/index.html` file in your browser then you will see the coverages of every file/component and you can navigate into each file to see where coverage is missing.

Looks like this:
![Screenshot 2023-09-26 at 8 47 38 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/8332986/eea2d337-95cb-41de-ac18-79f2b59add75)
